### ISSUE TITLE
Try learning rate scheduling

### ISSUE THREAD

**Issue by krutovsky-danya** [Cattharine/product_owner_rl#62]
What to do:
1. Plot loss function. It should decay
2. The problem is: when loss on platau - gradient step is too big to go deeper
3. It is usefull to decrece lr to reduce loss function
